incident_response_checklist.md

```diff
@@ -32,9 +33,4 @@ For on-call engineers
     - Use the date and a summary of the incident as the title
     - Fill out a draft of the incident timeline, to the best of your ability
     - Include a list of people who were heavily involved in remediation and should be present at the post-mortem
-- <input type='checkbox'> **Create an epic to track post-mortem actions and move to "Needs Post-Mortem"**
```

Comment:
this looks fine as is now...but hopefully we'll be able to update this soon...

Reply:
that's the goal 😀